### ISSUE TITLE
XML Parsing Error: not well-formed

### ISSUE THREAD

**Issue by cyclops24** [gadicc/meteor-sitemaps#29]
Hi,
First thanks for your awesome package.
I try to use your package but my sitemap.xml shown this error in browser:
```
XML Parsing Error: not well-formed
Location: http://localhost:3000/sitemap.xml
Line Number 22, Column 60:    <loc>http://localhost:3000/pages/title?q=query&lang=</loc>
---------------------------------------------------------------------------------^
```
But my sitemap seems to work. When viewing the source of my sitemap, the XML needed for crawlers is displayed.

URL is: http://localhost:3000/pages/title?q=query&lang=

How to add escape special character like [this](http://stackoverflow.com/a/3268544/2232900).

**Comment by gadicc** [comment]
Thanks for your patience.  I'm surprised this hasn't come up before.  This is fixed in the just released v0.0.22.  Thanks for the detailed report and references.

**Comment by cyclops24** [comment]
Thanks for your attention man. I'm going to update this awesome package.
